client/disputes/evidence/file-upload.tsx

```diff
@@ -11,28 +12,28 @@ import {
 } from '@wordpress/components';
 import Gridicon from 'gridicons';
 
-export const FileUploadControl = ( props ) => {
-	const {
-		fileName,
-		field,
-		disabled,
-		isDone,
-		isLoading,
-		accept,
-		error,
-		onFileChange,
-		onFileRemove,
-		help,
-	} = props;
-	const hasError = error && 0 < error.length;
-	const getIcon = () => {
-		return (
-			<Gridicon
-				icon={ isDone && ! hasError ? 'checkmark' : 'add-outline' }
-				size={ 18 }
-			/>
-		);
-	};
+import type { DisputeFileUpload } from 'wcpay/types/disputes';
+
+export const FileUploadControl = ( {
+	field,
+	fileName,
+	disabled,
+	isDone,
+	isLoading,
+	accept,
+	error,
+	onFileChange,
+	onFileRemove,
+	help,
+}: DisputeFileUpload ): JSX.Element => {
+	const hasError = ( error && 0 < error.length ) || undefined;
```

Comment:
`|| false`

Reply:
Thanks, makes sense to be `false` if the type is `boolean`. :)

Reply:
:+1: